client-data/tools/download/download.js

```diff
@@ -0,0 +1,57 @@
+/**
+ *                        WHITEBOPHIR
+ *********************************************************
+ * @licstart  The following is the entire license notice for the
+ *  JavaScript code in this page.
+ *
+ * Copyright (C) 2020  Ophir LOJKINE
+ *
+ *
+ * The JavaScript code in this page is free software: you can
+ * redistribute it and/or modify it under the terms of the GNU
+ * General Public License (GNU GPL) as published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option)
+ * any later version.  The code is distributed WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU GPL for more details.
+ *
+ * As additional permission under GNU GPL version 3 section 7, you
+ * may distribute non-source (e.g., minimized or compacted) forms of
+ * that code without the copy of the GNU GPL normally required by
+ * section 4, provided you include this license notice and a URL
+ * through which recipients can access the Corresponding Source.
+ *
+ * @licend
+ */
+
+(function download() { //Code isolation
+
+    function downloadSVGFile(evt) {
+        var canvasCopy = document.getElementById('canvas').cloneNode(true);
+        var styleNode = document.createElement("style");
+        styleNode.innerHTML = "rect, ellipse { fill:none; } path, line {fill: none;stroke-linecap: round; stroke-linejoin: round;}";
```

Comment:
Isn't there a way to get the actual style used on the page ? 
Putting it there means that any change or addition has to happen here AND in the respective CSS files of the different tools.

Reply:
In the PR of @sents `document.stylesheets` is used. 
https://github.com/sents/whitebophir/blob/fea42297dfbf5e219be87989505f5d0c441d9680/client-data/js/board.js#L613-L620

I added it and named him as Co-Author

Reply:
another way would be to add svg specific styles to the SVG, because now we are also copying website css to the exported SVG.